chinese/chinese-tomato-egg.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Chinese Tomato Egg Stir-Fry</title>
+  </head>
+  <body>
+    <img
+      src="https://christieathome.com/wp-content/uploads/2020/10/Chinese-Tomato-Egg-Stir-Fry-11-300x300.jpg"
+      alt=""
+    />
+    <h1>Chinese Tomato Egg Stir-Fry</h1>
+    <p>
+      Chinese tomato egg stir-fry. Soft and silky eggs stir fried in a sweet
+      tangy tomato sauce with green onions. Ready in 15 minutes in one pan!
+    </p>
+    <table>
```

Comment:
For now tables are fine, for the future let's not use them unless you really want a table specifically. Usually we use CSS to get a table-like layout on our pages. Tables are a very oldschool element :) Not a problem here, but just a hint and sth to consider in the future